src/theme/DocItem/index.js

```diff
@@ -1,16 +1,46 @@
 import React from 'react';
 
+import DocPaginator from '@theme/DocPaginator';
 import Head from '@docusaurus/Head';
 import isInternalUrl from '@docusaurus/isInternalUrl';
 import useDocusaurusContext from '@docusaurus/useDocusaurusContext';
 import useBaseUrl from '@docusaurus/useBaseUrl';
 import useTOCHighlight from '@theme/hooks/useTOCHighlight';
 
 import {RightSidebar} from 'libra-docusaurus';
+import ArrowLeft from 'img/shared/arrow-left.svg';
+import ArrowRight from 'img/shared/arrow-right.svg';
 
 import classnames from 'classnames';
 import styles from './styles.module.css';
 
+const Pagination = ({metadata}) => {
```

Comment:
Please move pagination to a new component

Reply:
Good idea